src/main/proto/control.proto

```diff
@@ -68,6 +71,9 @@ message PlatformOutboundInstruction {
 
         /* Instruction to merge two Segments in a Tracking Event Processor */
         EventProcessorSegmentReference merge_event_processor_segment = 9;
+
+        /* This heartbeat is used by AxonFramework in order to check the connection is still alive*/
```

Comment:
Nit: I'd rephrase slightly by adding `... in order to check *if* the connection is...`.